config/env/development.js

```diff
@@ -18,4 +18,7 @@ module.exports = {
   // models: {
   //   connection: 'someMongodbServer'
   // }
+  blueprints: {
+    rest: true,
+  },
```

Comment:
Enabling blueprints in development so we can test more easily. This lets us request localhost:1337/match to get all data for all matches, which I'm using in testing. Saves us the trouble of implementing dev-only endpoints and generally makes it easier to see into the database while mocking